201-data-factory-ftp-hive-blob/azuredeploy.parameters.json

```diff
@@ -3,13 +3,13 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "ftpHost": {
-      "value": "FTP server name or IP address"
+      "value": ""
```

Comment:
remove the entire parameter - empty string is still a valid value for AzureRM (and will fail in this sample) - e.g. remove lines 5-7 (and then for each param)